### PR TITLE
Joiner ESC key steps and Actions added and Code formatting -( Some changes were reverted which occured due to code formatting)

### PR DESCRIPTION
**Pull request by rahuldash171**
Joiner ESC key steps and Actions added and  Code formatting

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 1]
```diff
@@ -1026,6 +1028,14 @@ public static void startReplicationPipeline() {
     ElementHelper.clickIfDisplayed(CdfPluginPropertiesLocators.start, ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
   }
 
+  /**
+   * Escape after selecting a multi-select dropdown value
+   */
+  public static void pressEscapeKey() {
```
Comment (poojantcs) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Add logger

File: src/main/java/stepsdesign/PipelineSteps.java [hunk 2]
```diff
@@ -170,6 +170,10 @@ public void validateMandatoryPropertyErrorFor(String property) {
     CdfPluginPropertiesActions.verifyRequiredPropertyHasNoValueErrorMessage(property);
     CdfPluginPropertiesActions.verifyPluginPropertyInlineErrorMessageColor(property);
   }
+  @Then("Press ESC key to close the joiner fields dropdown")
```
Comment (poojantcs) [inline src/main/java/stepsdesign/PipelineSteps.java]: Make it generic as the same action is applied for all the Plugin properties page

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java (hunk 1 above)
Comment (rahuldash171) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Added Logger

File: src/main/java/stepsdesign/PipelineSteps.java (hunk 2 above)
Comment (rahuldash171) [reply src/main/java/stepsdesign/PipelineSteps.java]: Made the statement generic

**Pull request merged**